src/components/Schedule/Constants.js

```diff
@@ -0,0 +1,10 @@
+export const HOUR_HEIGHT = 150
+export const EVENT_GAP = 5
+export const EVENT_WIDTH = 400
+
+export const EVENT_TYPES = {
+  "main": ["MAIN", "#FFFFFF"],
```

Comment:
is `MAIN: "#FFFFFF"` any better? Was thinking accessing indices of an event type was a lil weird. Alternative,
```
main: {
    label: "MAIN",
    colour: "#FFFFFF",
}
```

Reply:
I wanted to keep it easy to modify but yeah I agree an object would be nicer to work with